ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -0,0 +1,482 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { UnreachableCode } from '../../../../../resources/not_reached';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Silkie specify which puff to get behind in first Slipper Soap
+// TODO: Silkie specify where to point puff's tether
+// TODO: Silkie call puff to go to for safety
+// TODO: Silkie call when to go to boss for safety
+// TODO: Silkie cleanup timeline
+// TODO: Gladiator triggers and timeline to enrage
+// TODO: Shadowcaster triggers and timeline to enrage
+
+export interface Data extends RaidbossData {
+  suds?: string;
+  soapCounter: number;
+  beaterCounter: number;
+  mightCasts: (NetMatches['StartsUsing'])[];
+  mightDir?: string;
+  hasLingering?: boolean;
+  thunderousEchoPlayer?: string;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AnotherSildihnSubterrane,
+  timelineFile: 'another_sildihn_subterrane.txt',
+  initData: () => {
+    return {
+      soapCounter: 0,
+      beaterCounter: 0,
+      mightCasts: [],
+    };
+  },
+  triggers: [
+    // ---------------- Silkie ----------------
+    {
```

Comment:
```suggestion
    {
      id: 'ASildihn Soap\'s Up',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['775A', '777D'], source: 'Silkie', capture: false }),
      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'cardinals',
        },
      },
    },
    {
```

Reply:
NB: this suggestion includes ASS and ASS Savage IDs

Reply:
There is a fizzling Suds cast before this that determines the soaps up ability, but seeing as soaps up only casts once and it's always fizzling Suds, this trigger works.

Reply:
Those were my thoughts as well.